internal/aws/k8scommon/kubeletutil/kubeletclient.go

```diff
@@ -0,0 +1,92 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
license header need to be changed

Reply:
Sure. This is a work-in-progress PR. I will add more changes

Reply:
The added files are only for container insight use case. So it won't affect existing memory consumption because those components will be not enabled. The final goal is to upstream those changes to opentelemetry-contrib. But it might takes some time, so for now, we plan to vendor the code in AOC temporarily.